docs/nodes/validate/add-a-validator.md

```diff
@@ -74,156 +75,332 @@ Once your validator’s start time has passed, you will see the rewards it may e
 
 That’s it!
 
-## Add a validator with API calls
+## Add a validator with AvalancheJS
+
+We can also add a node to the validator set using [AvalancheJS](https://docs.avax.network/apis/avalanchejs/). To add a node the Fuji Network, we’ll examine the workflow below.
+
+### Getting started
+
+To use AvalancheJS, you can clone the repo:
+```zsh
+$ git clone https://github.com/ava-labs/avalanchejs.git
+```
+
+or add it to an existing project:
+
+```zsh
+$ yarn add --dev avalanche
+```
+
+For this tutorial we will use `ts-node` to run the example scripts directly from an AvalancheJS directory.
+
+### Fuji Workflow
+
+Open your AvalancheJS directory and select the [**```examples/platformvm```**](https://github.com/ava-labs/avalanchejs/tree/master/examples/platformvm) folder to view the source code for the examples scripts. 
+
+We will use our [**```buildAddValidatorTx.ts```**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildAddValidatorTx.ts) script to add a validator.
+
+##### Private Key
 
-We can also add a node to the validator set by making API calls to our node. To add a node the Primary Network, we’ll call [`platform.addValidator`](../../apis/avalanchego/apis/p-chain.md#platformaddvalidator).
+Locate this line in the file
 
-This method’s signature is:
+```js
+const privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`;
+```
+
+and replace this with a private key that you control.
```

Comment:
Maybe link to appropriate example for getting a pirvate key? https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/createKeypair.ts

Reply:
Please update based on the comment above.

Reply:
committed

https://github.com/ava-labs/avalanche-docs/blob/a8a9c63fe3e51b34e1bcfb0b8c2be41c3bcbf13b/docs/nodes/validate/add-a-validator.md?plain=1#L103